lib/feedsme/index.js

```diff
@@ -213,7 +293,7 @@ Feedsme.prototype.trigger = function trigger(env, data, done) {
  */
 Feedsme.prototype.publish = function publish(data, fn) {
```

Comment:
stylistic thing, you've converted a bunch of these functions to async..await style but some of the type is still on callbacks.

Reply:
@msluther i can opt to thenify these rather than thenify at the consumer level

Reply:
I like that as it makes this api consistent from the outside (even if it's not intended to be consumed that way). feels a bit cleaner, but also I wouldn't block on it either. /shrug